public/wp-config-sample.php

```diff
@@ -64,14 +64,6 @@
  * The same with the core WordPress files.
  */
 
-define('WP_ENVIRONMENT', 'Development');
-
-if (WP_ENVIRONMENT == 'Production') {
-	define('WP_ROOT', '');
```

Comment:
I think we need to remove `WP_ENVIRONMENT`. Not `WP_ROOT`.

Make sure you still have this line in the code:
```php
define('WP_ROOT', '');
```

Reply:
Made changes in the latest commit.